src/modules/CoreUnit/resolvers.ts

```diff
@@ -2,14 +2,15 @@ import { stringArg, list, queryField } from 'nexus/dist';
 
 export const coreUnits = queryField('coreUnits', {
   type: list('CoreUnit'),
-  resolve: (parent, args, ctx) => ctx.prisma.coreUnit.findMany(),
+  resolve: (_parent, _args, ctx) => ctx.prisma.coreUnit.findMany(),
 });
 
 export const coreUnit = queryField('coreUnit', {
   type: 'CoreUnit',
   args: { id: stringArg() },
-  resolve: (parent, { id }, ctx) => {
+  resolve: (_parent, { id }, ctx) => {
     if (!id) {
+      ctx.request.req.log.error('coreUnit resolver: id is undefined');
       throw new Error('please provide id');
```

Comment:
- Wouldn't `please provide id` already be in the logs?
- I would suggest to add a log to every resolver (or do they already produce logs as well?)

Reply:
> Wouldn't please provide id already be in the logs?

The error that is thrown here gets captured and propagated to the api response.
Api response will contain all necessary info including traces (in case the env is `development`)

> I would suggest to add a log to every resolver (or do they already produce logs as well?)

there is only a single resolver apart from this one which directly uses prisma which in its turn already logs the information about the extraction. Should i still log the message with the response that is being sent out? i assume yes.

Reply:
well, i've also updated the package of apollo from deprecated one.